app/backend/controllers/article.js

```diff
@@ -0,0 +1,188 @@
+/*
+  Post method for articles.
+  It saves article to database.
+*/
+module.exports.postArticle = async (request, response) => {
+    let Article = request.models['Article']
+  
+    // Article instance to add to the database
+    let article = new Article({
+      ...request.body,
+      userId: request.session['user']._id,
+      date: new Date()
+    });
+  
+    // Saves the instance into the database, returns any error occured
+    article.save()
+      .then(doc => {
+      return response.status(204).send();
+    }).catch(error => {
+      return response.status(400).send(error);
+    });
+  }
+  
+  /*
+    Get method for articles.
+    It returns given article.
+  */
+  module.exports.getArticle = async (request, response) => {
+    let Article = request.models['Article']
+
+    article = await Article.findOne({ _id : request.params['id']});
```

Comment:
we also return owners name and surname as we did in followers. you can checkout utils.js